data/abilities.ts

```diff
@@ -4607,12 +4607,7 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 			if (pokemon.syrupTriggered) return;
 			pokemon.syrupTriggered = true;
 			this.add('-ability', pokemon, 'Supersweet Syrup');
-			let activated = false;
 			for (const target of pokemon.adjacentFoes()) {
-				if (!activated) {
-					this.add('-ability', pokemon, 'Supersweet Syrup', 'boost');
-					activated = true;
-				}
```

Comment:
This does not appear to be a documented change.

Reply:
Sounds like it needs to be rebased/merged, then? I can do that for you if that's difficult.